scripts/bootstrap

```diff
@@ -1,19 +1,10 @@
 #!/bin/bash
 
-set -e
-
-if [[ -n "${ECHOLOCATOR_DEBUG}" ]]; then
-    set -x
-fi
-
 DIR="$(dirname "${0}")/../"
+USAGE="Pull down secrets from S3."
```

Comment:
Since this is run by `setup`, which a dev is instructed to run in the README, should the README also include a reference to `ECHOLOCATOR_SETTINGS_BUCKET`? Without that, the `.env.sample` file gets copied to `.env`, instead of the real one being pulled down from S3.

Reply:
I had to [go back in history to check myself ](https://github.com/azavea/echo-locator/pull/500#discussion_r872517685)but I think that was the original intention, we want new dev users to copy `.env.sample` to `.env` and manually populate the values from the entry in 1password with the `ECHOLOCATOR_SETTINGS_BUCKET` mostly for loading the dotenv file in CI. However that feels clunky, especially when it could just pull from s3 and then go right to building. I checked and there *is* a `echo-locator-devdjango-config-us-east-1` bucket with an `.env` file that has the correct values for local development. I'll update the README to include `ECHOLOCATOR_SETTINGS_BUCKET` set to that value on initial spin up.

Reply:
Added README change in [d250ccd](https://github.com/azavea/echo-locator/pull/604/commits/d250ccd5b1f396d9cac5525ef6af23d6bf2e3d10)